lib/ulid/identifier.rb

```diff
@@ -41,12 +42,12 @@ def initialize(start = nil, seed = nil)
 
         # parse string into bytes
         @ulid = start.upcase
-        @bytes = decode(@ulid)
+        @bytes = decode32(@ulid)
 
         @time, @seed = unpack_decoded_bytes(@bytes)
       else
         # unrecognized initial values type given, just generate fresh ULID
-        @time = Time.now.utc
```

Comment:
Why remove the `.utc`?

Reply:
unnecessary since all the time conversion is done in epoch time